raphtory/src/serialise/parquet/mod.rs

```diff
@@ -279,17 +379,21 @@ fn collect_prop_columns(
                 });
             Ok((cols, graph_type))
         };
+
     let mut prop_columns = vec![];
     let mut g_type: Option<GraphType> = None;
-    for path in ls_parquet_files(path)? {
+
+    // Collect columns from just the first file
+    if let Some(path) = ls_parquet_files(path)?.next() {
         let (columns, tpe) = prop_columns_fn(&path, exclude)?;
+
         if g_type.is_none() {
             g_type = tpe;
         }
+
         prop_columns.extend_from_slice(&columns);
     }
-    prop_columns.sort();
```

Comment:
The reason why `test_ordered_dedupe_temporal_properties` was failing was because the prop columns would get sorted here, scrambling the original order they were inserted in.

IIUC, we can just infer the prop columns from the first parquet chunk file we see, instead of collecting columns from all the file and then deduping them. cc @fabianmurariu lmk if it's ok to remove the sort + dedup.

Reply:
If this turns out to be a problem we could always use an IndexSet to do the merge without scrambling the order, though I don't think it should be a problem typically.